glslc/src/file_compiler.h

```diff
@@ -69,12 +72,45 @@ class FileCompiler : public shaderc_util::Compiler {
   void SetIndividualCompilationMode();
 
   // Instead of outputting object files, output the disassembled textual output.
-  virtual void SetDisassemblyMode() override;
+  virtual void SetDisassemblyMode();
```

Comment:
Drop `virtual` and re-add `override`.

https://google.github.io/styleguide/cppguide.html#Inheritance